ONSdigital/dp-frontend-router
# remove CookiesRoutesEnabled flag

### What

Removed `CookiesRoutesEnabled` flag as GDPR is now live

### How to review

- Ensure relevant GDPR code is removed and that the cookies banner / preference page still load
- Check this in tandem with [this PR for `dp-frontend-renderer`](https://github.com/ONSdigital/dp-frontend-renderer/pull/508) and [this PR for `Babbage`](https://github.com/ONSdigital/babbage/pull/502)

### Who can review

Anyone
